app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  before_action :set_ticket, only: %i[show update]
+  before_action :set_participating_seller, only: %i[create]
+
+  # GET /tickets
+  def index
+    @tickets = Ticket.all
+
+    render json: @tickets
+  end
+
+  # GET /tickets/1
+  def show
+    render json: @ticket
+  end
+
+  # POST /tickets
+  def create
+    puts(params[:number_of_tickets])
+    savedCorrectly = true
+    createdTickets = []
+
+    (1..params[:number_of_tickets]).each do |_i|
```

Comment:
Is this really inclusive?

Reply:
Apparently **The two dot form creates an inclusive range, and the three-dot form creates a range that excludes the specified high value** I feel like starting at 1 and using the inclusive range makes a little more sense here? But happy to change it.

http://rubylearning.com/satishtalim/ruby_ranges.html#:~:text=The%20two%20dot%20form%20creates,references%20to%20two%20Fixnum%20objects.

Reply:
LOL wow this is so bad Ruby why